### ISSUE TITLE
Improve lookup for words with prefixes/suffixes

### ISSUE THREAD

**Issue by antistic** [openstenoproject/plover#1363]
**Is your feature request related to a problem? Please describe.**

I'd like to see ways of making words from prefixes/suffixes, even when these entries are not explicitly defined. Most of the time this is just a nice-to-have feature, but for some uncommon words the only defined entries could be thought of as misstrokes.

**Describe the solution you'd like**

The lookup window to show ways of making that translation from combinations of prefixes/word parts/suffixes, even if it is not explicitly defined in the dictionary.

**Describe alternatives you've considered**

- Could go in a plugin like spectra lexer instead. I'd prefer it in the lookup window mostly because it's what new people are most likely to use, and they are the people most likely to have difficulty working out how to write with prefixes/suffixes.
- Could update the dictionary to include these entries. This would work with words, but may not work well with, say, alternative spellings of names.

**Additional context**

It's come up several times in conversation on the discord server, but I can't seem to find an issue here already.

**Comment by user202729** [comment]
With the orthography rules, the problem may be very hard to impossible (to solve efficiently).

Besides, it will have a lot of "wrong" entries if someone decide to, for example, have a prefix/suffix for every character in the alphabet.

**Comment by antistic** [comment]
Yeah, this is something that jorwat on the discord has called the `KEUT/SKWRE` kite problem.

It could be possible to limit this to "common" prefixes and suffixes. The idea being that this is a tool for beginners, and more experienced stenographers would have more the skills to discern a bad entry from a good one.

This problem is closely linked to the problem of misstrokes in the dictionary (another thing that's not good for beginners, but useful for the more experienced), and the problem of good documentation and teaching materials (which don't talk about prefixes/suffixes much, especially near the beginning, and don't talk about Plover lookup window good practices at all).
